src/component/Error/Error.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+
+interface Props {
+  error: string,
+  onClose: () => void,
+}
+
+export const Error: React.FC<Props> = ({
+  error,
+  onClose: handleCloseError,
+}) => {
+  useEffect(() => {
+    setTimeout(() => handleCloseError(), 3000);
```

Comment:
Let's make a `clearTimeout` when component was deleted
How we can do it in `useEffect`?